analyze_job_data.py

```diff
@@ -0,0 +1,32 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+import datetime
+import pytz
+
+def analyze_job_data(df):
+    # Load the data
+
+    # Get current date in UTC timezone
+    utc_now = datetime.datetime.now(datetime.UTC)
+
+    # Convert to a specific timezone (for example, 'America/New_York')
+    desired_timezone = pytz.timezone('America/Los_Angeles')
+    today_timezone_aware = utc_now.replace(tzinfo=pytz.utc).astimezone(desired_timezone)
+
+
+    df["Meta_DatePosted"] = pd.to_datetime(df["Meta_DatePosted"])
+    df['Listing_Duration'] = pd.to_datetime(today_timezone_aware) - df['Meta_DatePosted']
```

Comment:
nit: Just a formatting thing, but I would use consistent quotes for string across your code. Just need to pick one and stick to it for all your strings. You can use any auto-formatter like `ruff` or `black` for consistency in code formatting